### ISSUE TITLE
How to preview text-based file like 'go.mod"

### ISSUE THREAD

**Issue by programervn** [sbarex/SourceCodeSyntaxHighlight#117]
This app work great for source code file, 
How to handle other text-based file, for example, I use Go, and project always have go.mod file, but cannot previews
this file type, can I add this file type to preview?

<img width="617" alt="image" src="https://user-images.githubusercontent.com/3676829/139407949-816ff534-e4cc-43fd-b7be-7703c67987b5.png">

**Comment by sbarex** [comment]
I'm unsure whether to enable support for these files. MacOs identifies files based on their UTI which is derived from the file extension. To support these files, all files with `.mod` and `.sum` extension would be handled even if their name is not `go`.

**Comment by dufferzafar** [comment]
@sbarex I agree that having built-in support for extensions that may mean different things (`.mod` go file or a `.mod`)

Is it possible to extend the rule matching for files without extension to files even with extension - this would allow users to customise such behaviour for themselves.

**Comment by sbarex** [comment]
No, is not possible, see https://github.com/sbarex/SourceCodeSyntaxHighlight/issues/170#issuecomment-1186195750
